services/generic/battery.md

```diff
@@ -1,44 +1,68 @@
-### Battery service
+# Battery Service
 
-#### Service names
+Battery service represents a power unit within the device, usually found in battery-powered sensors, detectors, controllers, or similar devices.
+
+## Service names
 
 `battery`
 
-#### Interfaces
+## Interfaces
 
-Type | Interface          | Value type | Properties | Description
------|--------------------|------------|----------- |------------------
-in   | cmd.lvl.get_report | null       |            | Get battery level over level report.
-out  | evt.lvl.report     | int        | state      |
--|||
-out  | evt.alarm.report   | str_map    |            | val = {"event": "low_battery", "status": "activ"}
--|||
-in   | cmd.health.get_report| null     |            | Request battery health report
-out  | evt.health.report  | int        |            | Battery health in %
-in   | cmd.sensor.get_report | string  |            | Request battery temperature report in celsius.
-out  | evt.sensor.report  | float      | unit       | Battery temperature in celsius.
--|||
-in   | cmd.battery.get_report | null |              | Get full battery report
-out  | evt.battery.report | object |                | Battery full report
```

Comment:
So You checked that none of other non-adapter services uses those interfaces? :D

Reply:
Not a single adapter in production advertises these interfaces. If you have any information about it, please share it. I believe these additions were made in preparation for household battery support which never went past early PoC.